mono/eglib/glib.h

```diff
@@ -755,10 +751,7 @@ GUnicodeBreakType   g_unichar_break_type (gunichar c);
  * format must be a string literal, in order to be concatenated.
  * If this is too restrictive, g_error remains.
  */
-#define  g_assertf(x, format, ...) \
-	(G_LIKELY(x) ? 1 : (g_assertion_message ( \
-	"* Assertion at %s:%d, condition `%s' not met, function:%s, " format "\n", \
-	__FILE__, __LINE__, #x, __func__, __VA_ARGS__), 0))
+#define  g_assertf(x, format, ...) (G_LIKELY((x)) ? 1 : (g_assertion_message ("* Assertion at %s:%d, condition `%s' not met, function:%s, " format "\n", __FILE__, __LINE__, #x, __func__, ##__VA_ARGS__), 0))
```

Comment:
Double parens not needed, on G_LIKELY.

Reply:
`##__VA_ARGS__` is a gcc extension that's not supported by MSVC.